content/blog/2021-08-24-transfer-learning-experiments.md

```diff
@@ -14,6 +14,7 @@ tags:
   - Experiments
   - Reproducibility
   - DVC
+  - Pre-trained Models
```

Comment:
no worries, I was curious about the new post though - this tag can potentially also apply there (but we are reusing the model in the new post also?)

Reply:
I think since the focus of this post is on the data changes and not the pretrained model that it's ok to leave it off of the new post.